docs/Usage.md

```diff
@@ -85,36 +85,52 @@ Important points you should consider when training your networks with compressio
   - Turn off the `Dropout` layers (and similar ones like `DropConnect`) when training a network with quantization or sparsity
   - It is better to turn off additional regularization in the loss function (for example, L2 regularization via `weight_decay`) when training the network with RB sparsity, since it already imposes an L0 regularization term.
 
-#### Step 4 (optional): Export the compressed model to ONNX
-After the compressed model has been fine-tuned to acceptable accuracy and compression stages, you can export it to ONNX format.
-Since export process is in general algorithm-specific, you have to call the compression controller's `export_model` method to properly export the model with compression specifics into ONNX:
+#### Step 4: Export the compressed model
+After the compressed model has been fine-tuned to acceptable accuracy and compression stages, you can export it. There are two ways to export a model.
+
+First, you have to call the compression controller's `export_model` method to properly export the model with compression specifics into ONNX:
+
 ```python
 compression_ctrl.export_model("./compressed_model.onnx")
 ```
 The exported ONNX file may contain special, non-ONNX-standard operations and layers to leverage full compressed/low-precision potential of the OpenVINO toolkit.
 In some cases it is possible to export a compressed model with ONNX standard operations only (so that it can be run using `onnxruntime`, for example) - this is the case for the 8-bit symmetric quantization and sparsity/filter pruning algorithms.
 Refer to [compression algorithm documentation](./compression_algorithms) for details.
 
+Second, you have to call the compression controller's `prepare_for_inference` method, supports only for PyTorch, to properly get the model without NNCF specific nodes for training compressed model.
```

Comment:
Please mention here that `.prepare_for_...()` is more flexible since it allows using standard PyTorch export capabilities as well as plug 3rd party SW inference solutions that can work with PyTorch models.

Reply:
please check new version